test/data/corner_cases/c.html

```diff
@@ -0,0 +1 @@
+c#.html
```

Comment:
This comment applies to the name of the file.

Holding on to my previous concern wrt semantic consistency, I have to note that C is not C#. Maybe rename this file to c_sharp.html?

Reply:
Should be fixed now 👍